samples/powershell/GenerateErrorReport.ps1

```diff
@@ -1,19 +1,25 @@
 Try {
-    [System.Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+    [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.SecurityProtocolType]::Tls12
     Import-Module ((Split-Path -Parent $MyInvocation.MyCommand.Definition) + "\Common.ps1") -Force
     Import-Module -Name "Fly.Client"
+    #Get the global configuration object to set Fly_API_Endpoint and your access token
     $Configuration = Get-Configuration
     $Configuration["BaseUrl"] = "{Fly_API_Endpoint}"
     $Configuration.AccessToken = "YOUR_BEARER_TOKEN"
```

Comment:
How to get the bearer token？

Reply:
First we add comment in Fly user guide about how to get the bearer token, second we provide a single GetIdentityServiceToken.ps1 for customer, we will add comment to refer it